packages/labs/eslint-plugin/src/lib/util.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {AbsolutePath, Analyzer} from '@lit-labs/analyzer';
+import {
+  ESLintUtils,
+  ParserServicesWithTypeInformation,
+} from '@typescript-eslint/utils';
+// TODO (justinfagnani): get from ESLintUtils if possible?
+// See q: https://discord.com/channels/1026804805894672454/1084238921677946992/1199174067459198986
+import ts from 'typescript';
+import * as path from 'path';
+
+export const createRule = ESLintUtils.RuleCreator(
+  (name: string) => `https://lit.dev/eslint-plugin/${name}`
```

Comment:
is this worth a TODO or an issue for actually adding docs to lit.dev ?

Reply:
ah, yes